### PR TITLE
Added Python requirement to AAP docs

### PR DESCRIPTION
**Pull request by ianf77**
Added Python requirement to Upgrade/migration guide and
Installation Guide

Add python requirements for AAP into docs

https://issues.redhat.com/browse/AAP-3385

### CODE REVIEW

**Comment by ianf77** [comment]
Needs cherry-picking to 2.0, 2.1 and 2.2. Although changes to the Installation Guide might be irrelevant given the rewrite.

**Comment by ariordan-redhat** [comment]
@ianf77 FYI, 2.0 wasn't published. You will need to cherry-pick to the 2.0-ea branch instead.

**Comment by ianf77** [comment]
Sorry. Who does the cherry-picking?

**Comment by oraNod** [comment]
> Sorry. Who does the cherry-picking?

Personally I feel like the person who opens the PR is responsible for cherry-picking after merge then opening backport PRs. IMO that's the proper process.

**Comment by ariordan-redhat** [comment]
@RedHatInsights/aap-docs-pr-review-group

**Comment by ariordan-redhat** [comment]
@ianf77 Whoever opens the PR does the cherry-picking and backport PRs.

**Review by ariordan-redhat — APPROVED** [review]

File: downstream/modules/platform/ref-system-requirements.adoc [hunk 1]
```diff
@@ -24,6 +24,8 @@ h| Subscription | Valid Red Hat Ansible Automation Platform |
 h| OS | Red Hat Enterprise Linux 8.4 or later 64-bit (x86) |
 
 h| Ansible | version 2.11 required | If Ansible is not already present on the system, the setup playbook will install `ansible-core` 2.12.
+
+h| Python | Python 3.8 or later |
```
Comment (oraNod) [inline downstream/modules/platform/ref-system-requirements.adoc:28]:
```suggestion
h| Python | 3.8 or later |
```

**Review by oraNod — APPROVED** [review]

**Comment by ariordan-redhat** [comment]
Technically, this PR hasn't been backported to `2.0-ea`.
Copied only the change to `/modules/platform/ref-system-requirements.adoc` to the `2.0-ea` branch.
The `/modules/platform/ref-system-requirements.adoc` file in the `2.1` branch is different to its `2.0-ea` counterpart.
The other files in this PR do not exist in `2.0-ea`.

**Comment by oraNod** [comment]
@ariordan-redhat Out of curiousity is the content on `2.0-ea` hooked into the publishing system or is that kind of a dead branch? I don't even see a 2.0 version on the splash page.

**Pull request merged**